### ISSUE TITLE
Outdated dependency

### ISSUE THREAD

**Issue by saropa** [fernan542/fl_country_code_picker#3]
Hi, is it possible to bump up the dependencies?
            
```
Because fl_country_code_picker 0.0.3 depends on scrollable_positioned_list ^0.2.2 and no versions of fl_country_code_picker match >0.0.3 <0.1.0, fl_country_code_picker ^0.0.3 requires scrollable_positioned_list ^0.2.2.
So, because _[app]_ depends on both fl_country_code_picker ^0.0.3 and scrollable_positioned_list ^0.3.4, version solving failed.
pub get failed (1; So, because  _[app]_ depends on both fl_country_code_picker ^0.0.3 and scrollable_positioned_list ^0.3.4, version solving failed.)
exit code 1
```

**Comment by fernan542** [comment]
Hello @saropa, sure!

Thanks for your interest in my package.

**Comment by fernan542** [comment]
Implemented at version `0.0.3+1`.
Feel free to open this again if an error in dependencies still occur. 
Many thanks!
